tests/unittests/Caffe2ImporterTest.cpp

```diff
@@ -605,16 +605,47 @@ TEST(caffe2, FC) {
     updateInputPlaceholdersByName(ctx, &mod, {"inputs"}, {&inputs});
   }
 
-  EE.compile(CompilationMode::Infer, F);
-  EE.run(ctx);
+  // High level check on the content of the graph. We have 1 FC node and 1 save.
+  EXPECT_EQ(F->getNodes().size(), 2);
+  auto *saveNode = getSaveNodeFromDest(output);
+  auto *fcNode =
+      llvm::dyn_cast<FullyConnectedNode>(saveNode->getInput().getNode());
+  ASSERT_TRUE(fcNode);
```

Comment:
I *think* we can get away with an `EXPECT_TRUE` here since we don't dereference `fcNode` after this.

Reply:
Changed to `EXPECT_TRUE`.